Sources/_TestingInternals/CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ if("${CMAKE_CXX_COMPILER_FRONTEND_VARIANT}" STREQUAL "MSVC" OR
 elseif(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
   target_compile_options(_TestingInternals PRIVATE
     -fno-exceptions -fPIC)
+elseif(CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
+  target_compile_options(_TestingInternals PRIVATE
+    -fno-exceptions
+    -DSWT_TESTING_LIBRARY_VERSION="${SWT_VERSION_FILE_CONTENTS}")
```

Comment:
Looks like you moved the definition of `${SWT_VERSION_FILE_CONTENTS}` to CompilerSettings.cmake, but not that of `SWT_TESTING_LIBRARY_VERSION`.

What' I'd ask you do here is:

- Remove this `-D` line from this file, as it's inconsistent with how we define other C++ macros
- Add an equivalent `add_compile_definitions()` call in CompilerSettings.cmake after the call to `file(STRINGS)`
- And this last one is nitpicky but call the intermediate CMake value `${SWT_TESTING_LIBRARY_VERSION}` instead of `${SWT_VERSION_FILE_CONTENTS}` for consistency and so it's clear they refer to the same value.

Thanks much! :)